### ISSUE TITLE
gquic:replace aes12 to standard aes lib

### ISSUE THREAD

**Issue by kordenlu** [lucas-clemente/quic-go#1877]
the standard lib crypto/cipher  applies an export method cipher.NewGCMWithTagSize, we can replace aes12 with standard lib now

**Comment by marten-seemann** [comment]
Cool, I didn't see that they added support for that. Was in Go 1.11 already, apparently.
Would you mind sending us a pull request with this change?

**Comment by kordenlu** [comment]
yeah，NewGCMWithTagSize was introduced since GO 1.11, here is the fix record: https://github.com/golang/go/issues/24977

it's my pleasures to pull request for it,  i will take a try this week.

still, i find another problem here, the 0RTT message always failed to decrypt. errors show as below
~~~
2019/04/16 20:56:24 server forwardSecureAEAD failed,packet:2,err:cipher: message authentication failed
2019/04/16 20:56:24 server secureAEAD failed,packet:2,err:cipher: message authentication failed
2019/04/16 20:56:24 server <- Reading packet 0x2 (325 bytes) for connection 0xeab6b388867831f7 err:DecryptionFailure: NullAEAD: failed to authenticate received data ([]byte{0xe6, 0x9e, 0xe3, 0x7a, 0x7, 0x9d, 0x41, 0x26, 0x8e, 0x4f, 0x39, 0x2b} vs []byte{0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0})
~~~

**Comment by marten-seemann** [comment]
Fixed by #1879.
